Sources/PrimerSDK/Classes/Core/PrimerHeadlessUniversalCheckout/Models/PrimerCardValidationModels.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  PrimerCardValidationModels.swift
+//  PrimerSDK
+//
+//  Created by Jack Newcombe on 30/10/2023.
+//
+
+import Foundation
+
+@objc
+public class PrimerCardValidationState: NSObject {
```

Comment:
Should we consider calling this `PrimerCardState` instead of `PrimerCardValidationState`? As we are not validating  anything here

Reply:
Yeah - I actually call the argument `cardState` in most places, so this makes sense. Will change 👍